ansible/roles/dev-desktop/tasks/main.yml

```diff
@@ -209,8 +218,18 @@
       - valgrind
       - ninja-build
       - fish
+      - ripgrep
+      - jq
+      - powershell
     state: present
 
+- name: Install git-delta
+  shell: |
+    DELTA_VERSION=$(curl --silent https://api.github.com/repos/dandavison/delta/releases/latest | jq --raw-output '.name')
+    curl -O git-delta.deb https://github.com/dandavison/delta/releases/download/$DELTA_VERSION/git-delta_$DELTA_VERSION_$(uname -m).deb
+    apt install ./git-delta.deb
+    rm ./git-delta.deb
```

Comment:
admittedly this is kinda janky; I'm ok with installing it just for my user if you don't think other people on the machine will use it.

Reply:
👍 makes sense, I don't think a lot of other people are using it, happy to remove it